pydantic_ai_slim/pydantic_ai/template.py

```
@@ -0,0 +1,135 @@
+        class MyDeps:
+            name: str
+
+        agent = Agent(
+            'openai:gpt-5',
+            deps_type=MyDeps,
+            instructions=TemplateStr('Hello {{name}}'),
+        )
+        ```
+    """
+
+    __slots__ = ('_source', '_deps_type', '_deps_schema', '_compiled_typed', '_compiled_untyped')
```

Ruff (0.15.21)

[warning] `TemplateStr.__slots__` is not sorted

Apply a natural sort to `TemplateStr.__slots__`

(RUF023)